config/initializers/teaspoon.rb

```diff
@@ -0,0 +1,7 @@
+unless Rails.env.production?
+  module Teaspoon
+    class SuiteController < ActionController::Base
```

Comment:
I don't really understand what needs to happen here. As you've seen, my rails and ruby knowledge is really rudimentary. It seems to work as in, but this will cause issues in some way down the line?